catalog/transformer/catalog-transformer-propertyjson-metacard/src/main/java/ddf/catalog/transformer/metacard/propertyjson/PropertyJsonMetacardTransformer.java

```diff
@@ -181,6 +181,9 @@ private static Object convertValue(Serializable value, AttributeType.AttributeFo
 
     switch (format) {
       case DATE:
+        if (!(value instanceof Date)) {
+          return null;
```

Comment:
:question: Should we be logging something at debug or trace that we are dropping the date?

Reply:
Wouldn't hurt. Will log a debug message here